velox/exec/Unnest.cpp

```diff
@@ -268,17 +252,25 @@ VectorPtr Unnest::generateOrdinalityVector(const RowRange& range) {
 
   VELOX_DCHECK_GT(range.size, 0);
 
+  vector_size_t index = 0;
   range.forEachRow(
-      [&](vector_size_t /*row*/, vector_size_t start, vector_size_t size) {
+      [&](vector_size_t row, vector_size_t start, vector_size_t size) {
+        if constexpr(isOuter) {
+          if (rawOrdinalityIsNull_[row]) {
+            ordinalityVector->setNull(index, true);
+          }
+        }
         std::iota(rawOrdinality, rawOrdinality + size, start + 1);
         rawOrdinality += size;
```

Comment:
Then we don't need to move rawOrdinality, just use `rawOrdinality + index` in L263.

Reply:
you mean change it to `std::iota(rawOrdinality + index, rawOrdinality + index + size, start + 1);` right?

Reply:
No,
```
index += size;
std::iota(rawOrdinality, rawOrdinality +index, start + 1);
```

Reply:
And drop `rawOrdinality += size;`

Reply:
Place the index += size prior to `std::iota`, the semantic is equal
```
index += size;
std::iota(rawOrdinality, rawOrdinality +index, start + 1);
```

Reply:
Yes, you are right, please keep current approach